### PR TITLE
Initial iam permissions to allow for cloudtruth push feature

### PR DESCRIPTION
**Pull request by wr0ngway**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -68,6 +81,7 @@ data "aws_iam_policy_document" "ssm" {
   statement {
     sid = "ParameterAccess"
     actions = [
+      "ssm:DescribeParameters",
       "ssm:GetParameter",
       "ssm:GetParameters",
```
Comment (jeking3) [inline main.tf:86]: We do not call `get_parameters`.

File: main.tf [hunk 2]
```diff
@@ -68,6 +81,7 @@ data "aws_iam_policy_document" "ssm" {
...
       "ssm:GetParametersByPath"
```
Comment (jeking3) [inline main.tf:87]: We do not call `get_parameters_by_path`.

**Review by jeking3 — APPROVED** [review]
It looks like we have a couple SSM read permissions we do not need (not part of this changeset).  Otherwise looks like the correct set.

File: main.tf (hunk 2 above)
Comment (wr0ngway) [reply main.tf:87]: Since the level of read they grant is similar, I'm going to leave them in for now in case we want to pull params in bulk in the future

**Pull request merged**